tests/test_threadpoolctl.py

```diff
@@ -313,7 +314,6 @@ def test_get_original_num_threads(limit):
         original_info = _threadpool_info()
         with threadpool_limits(limits=limit, user_api="blas") as threadpoolctx:
             original_num_threads = threadpoolctx.get_original_num_threads()
-            print(original_num_threads)
```

Comment:
It would be great to find a way to get pytest to fail if any test prints anything to stdout / stderr. But don't worry if this is not easy to implement. It's just a nice to have.

Reply:
I only found a way to check the stdout/stderr within each test. Not a reasonable solution...